pkgs/development/python-modules/pandoc/default.nix

```
@@ -0,0 +1,37 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, plumbum
+, ply
+}:
+
+buildPythonPackage rec {
+  pname = "pandoc";
+  version = "2.3";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "boisgera";
+    repo = pname;
```

```suggestion
    repo = "pandoc";
```

See https://github.com/NixOS/nixpkgs/issues/277994